modulemd/v2/modulemd-component-rpm.c

```diff
@@ -291,6 +291,80 @@ modulemd_component_rpm_set_property (GObject *object,
 }
 
 
+gboolean
+modulemd_component_rpm_equals (ModulemdComponentRpm *self,
```

Comment:
This function needs to chain up to the `klass->equals()` function of the parent, or it will not be checking buildorder, name or rationale.

See [`modulemd_component_module_copy()`](https://github.com/fedora-modularity/libmodulemd/blob/9d4d6f5a7a1f45c94062117ade0e30c665bef052/modulemd/v2/modulemd-component-module.c#L67) for an example of how to do this.

Reply:
Same problem as above.